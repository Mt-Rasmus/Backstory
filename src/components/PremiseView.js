
import React from 'react';

const PremiseView = (props) => {
   return (
      <div>
         {props.premise}
         <button 
            className="button button--standard button-modal" 
            onClick={props.toggleShowBackstory}>
            Backstory
         </button>         
      </div>
   )

}

export default PremiseView