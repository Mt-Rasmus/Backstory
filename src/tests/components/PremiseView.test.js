
import React from 'react';
import PremiseView from '../../components/PremiseView';
import { shallow } from 'enzyme';

test('Should render PremiseView correctly', () => {
   const wrapper = shallow(<PremiseView />);
   expect(wrapper).toMatchSnapshot();
});
