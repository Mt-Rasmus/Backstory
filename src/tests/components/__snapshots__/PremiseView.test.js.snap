// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render PremiseView correctly 1`] = `
<div>
  <button
    className="button button--standard button-modal"
  >
    Backstory
  </button>
</div>
`;
